### PR TITLE
Bibek add ee code

### PR DESCRIPTION
**Pull request by Aryal007**

### CODE REVIEW

File: conf/readme.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+data:
```
Comment (krisrs1128) [inline conf/readme.yaml]: Did you intend to include all these .yaml configuration files? I feel like the one we have for defaults is already enough, but maybe there are some use cases I haven't thought of.

File: explore-lr-experiment--1/run_0/run-explore-lr.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```
Comment (krisrs1128) [inline explore-lr-experiment--1/run_0/run-explore-lr.sh]: I also am not sure whether these run scripts are really necessary. They're automatically generated by the parallel run script, so it seems a little redundant.

File: conf/readme.yaml (hunk 1 above)
Comment (Aryal007) [reply conf/readme.yaml]: These were unintended, Thank you. I will delete the files.

File: explore-lr-experiment--1/run_0/run-explore-lr.sh (hunk 2 above)
Comment (Aryal007) [reply explore-lr-experiment--1/run_0/run-explore-lr.sh]: These were unintended. Will delete them, thanks

**Pull request merged**